src/vstt/stats.py

```diff
@@ -221,6 +248,67 @@ def stats_dataframe(trial_handler: TrialHandlerExt) -> pd.DataFrame:
         ),
         axis=1,
     )
+    df["movement_time_at_peak_velocity"] = df.apply(
+        lambda x: _movement_time_at_peak_velocity(
+            np.concatenate((x["to_target_timestamps"], x["to_center_timestamps"])),
+            np.concatenate(
+                (
+                    x["to_target_mouse_positions"],
+                    x["to_center_mouse_positions"].reshape(
+                        x["to_center_mouse_positions"].shape[0], 2
+                    ),
+                )
+            ),
+            x["to_target_num_timestamps_before_visible"],
+        ),
+        axis=1,
+    )
+    df["total_time_at_peak_velocity"] = df.apply(
+        lambda x: _total_time_at_peak_velocity(
+            np.concatenate((x["to_target_timestamps"], x["to_center_timestamps"])),
+            np.concatenate(
+                (
+                    x["to_target_mouse_positions"],
+                    x["to_center_mouse_positions"].reshape(
+                        x["to_center_mouse_positions"].shape[0], 2
```

Comment:
this is repeated a few times, may be better to refactor into a `concatenate_positions()` function? also there is already `preprocess_mouse_positions()` which looks like it does something similar